Adyen/client.py

```diff
@@ -492,7 +496,8 @@ def call_checkout_api(self, request_data, action, idempotency_key=None,
             "authorise3ds2",
             "payments",
             "paymentSession",
-            "paymentLinks"
+            "paymentLinks",
+            "paymentMethodsBalance"
```

Comment:
should we add `orders` and `ordersCancel` here or it's not supported?

Reply:
Checking the documentation the application info is only part of paymentMethodsBalance request.